git_utils.py

```diff
@@ -165,7 +165,7 @@ def gitGetRemoteTags( url, debug = False, verbose = False ):
     try:
         if verbose:
             print("gitGetRemoteTags running: git ls-remote %s" % url)
-        statusInfo = subprocess.check_output( [ 'git', 'ls-remote', url ], stderr=subprocess.STDOUT, universal_newlines=True )
```

Comment:
Is universal_newlines removed on purpose here?

Reply:
Yep, `text=True` accomplishes the same thing